aten/src/ATen/native/native_functions.yaml

```diff
@@ -6711,16 +6711,20 @@
 - func: to_dense(Tensor self, ScalarType? dtype=None) -> Tensor
   variants: method
 
+# In to_dense.masked, the masked argument appears before dtype to avoid ambiguity with to_dense
```

Comment:
Is there precedent for adding these types of overloads? Are they for the benefit of the C++ API  where named arguments do not exist?

Reply:
We don't really need an overload here. We could just add a keyword-only argument with default value `True` to the existing signature.

Reply:
@cpuhrsch, my initial approach used the keyword-only argument approach and it did work out for all tests except for test/jit/test_freezing.py, see https://pytorch.slack.com/archives/CFZ7V4DNZ/p1678140643591249 .

Reply:
Not using overloads anymore, so resolving.